app/views/idv/doc_auth/send_link.html.erb

```diff
@@ -16,7 +16,8 @@
 
 <p><%= t('doc_auth.instructions.send_sms') %></p>
 <%= simple_form_for(
-      :doc_auth,
+      idv_phone_form,
```

Comment:
just to check: the only reason we're building the form is so we can pass it something that responds to `#phone` right? Could we pass a different object or like a simple PORO/Struct that has a `#phone` method?

Reply:
I based this usage on `Idv::PhoneController` which _also_ uses `Idv::PhoneForm`.  The form has some logic already built into it to pre-fill the user's MFA phone number.